### PR TITLE
Add bioasq/signal-1m/robust04/trec-news flat baselines

### PR DESCRIPTION
**Pull request by justram**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -114,6 +114,10 @@ See individual pages for details!
   + SciFact: ["flat" baseline](docs/regressions-beir-v1.0.0-scifact-flat.md), [SPLADE-distill CoCodenser-medium](docs/regressions-beir-v1.0.0-scifact-splade-distil-cocodenser-medium.md)
   + TREC-COVID: ["flat" baseline](docs/regressions-beir-v1.0.0-trec-covid-flat.md), [SPLADE-distill CoCodenser-medium](docs/regressions-beir-v1.0.0-trec-covid-splade-distil-cocodenser-medium.md)
   + Touche2020: ["flat" baseline](docs/regressions-beir-v1.0.0-webis-touche2020-flat.md), [SPLADE-distill CoCodenser-medium](docs/regressions-beir-v1.0.0-webis-touche2020-splade-distil-cocodenser-medium.md)
+  + BioASQ: ["flat" baseline](docs/regressions-beir-v1.0.0-bioasq-flat.md)
```
Comment (lintool) [inline README.md:117]: Should we reorder the corpora alphabetically? cc/ @NThakur20

**Review by lintool — APPROVED** [review]
lg, pending ordering of corpora on main README page.

File: docs/regressions-beir-v1.0.0-bioasq-flat.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
+# Anserini Regressions: BEIR (v1.0.0) &mdash; bioasq
```
Comment (lintool) [inline docs/regressions-beir-v1.0.0-bioasq-flat.md:1]: Also, @justram - should we go through each of the headings and add the correct capitalization of each corpus? E.g., BioASQ not bioasq?

File: README.md (hunk 1 above)
Comment (justram) [reply README.md:117]: I'll merge this PR first for @NThakur20 's next multifield baseline PR.

**Pull request merged**